pages/security.md

```diff
@@ -91,13 +91,14 @@ If you want to use Keycloak with Docker Compose, be sure to read our [Docker Com
 
 > <i class="fa fa-info-circle"></i> **Note for JHipster 7.8.1 and Keycloak 16.1.0 on Apple Silicon (M1)**
 > 
-> Keycloak may misbehave on Apple Silicon in compatability mode and the solution is not obvious. You may want to build the Keycloak image locally to address. The following steps are known to work:
+> Keycloak may misbehave on Apple Silicon in compatability mode and the solution is not obvious. You can build the Keycloak image locally to solve the problem:
 > 
-> 1. Clone Keycloak containers repository: `git clone git@github.com:keycloak/keycloak-containers.git`
-> 1. Open `server` directory, e.g., `cd keycloak-containers/server`
-> 1. Checkout at desired version, e.g., `git checkout 16.1.0`
-> 1. Build docker image `docker build -t jboss/keycloak:16.1.0 .`
-> 1. Run Keycloak `npm run docker:keycloak:up`
+> ```
+> git clone git@github.com:keycloak/keycloak-containers.git
+> cd keycloak-containers/server
+> git checkout 16.1.0
```

Comment:
Maybe use v19, to align with the version from generator-jhipster : https://github.com/jhipster/generator-jhipster/blob/main/generators/generator-constants.js#L78-L79

Reply:
This only applies to JHipster 7.8.1 and the version of Keycloak that shipped with it. It hasn't been necessary since JHipster 7.9.0.

I'm fine with removing it too. 🙂